includes/phing/src/Tasks/PlatformVersionsTask.php

```diff
@@ -200,7 +200,25 @@ private function callGithubReleases($url)
             curl_setopt($curl, CURLOPT_HTTPHEADER, $request_headers);
         }
         $resp = json_decode(curl_exec($curl));
+        $http_code = curl_getinfo($curl, CURLINFO_HTTP_CODE);
         curl_close($curl);
+        if ($http_code !== 200) {
```

Comment:
No need to call `json_decode()` if result is other than 200. Maybe we should move validation to before.

Reply:
There is a need for `json_decode()` since I'm using the $resp->message to be displayed in the log.